backend/organization/generate_organizations.py

```diff
@@ -21,14 +21,14 @@ def generate(orgs):
 
     for org in orgs:
         # create intern organization (request from worker A to backend A)
-        secret = generate_secret()
+        secret = generate_password()
         files[org]["outgoing_organizations"].append({"organization_id": org, "secret": secret})
         files[org]["incoming_organizations"].append({"organization_id": org, "secret": secret})
```

Comment:
```suggestion
        files[org]["incoming_organizations"].append({"organization_id": org, "secret": secret})
```
Why is it secret here if it is a password in fact ?

Reply:
Changed ! althought didn't change on `OutgoingOrganization`, as the DB column `secret` is actually password and it may become something bigger than this issue.